Backend/mentii/class_ctrl.py

```diff
@@ -8,26 +8,11 @@
 
 def getActiveClassList(dynamoDBInstance, email=None):
   response = ControllerResponse()
-  if email is None:
-    email = g.authenticatedUser['email']
-  dynamodb = dynamoDBInstance
-  table = dbUtils.getTable('users', dynamodb)
-  if table is None:
-    MentiiLogging.getLogger().error("Unable to get users table in getActiveClassList")
-    return None
-  classCodes = []
   classes = []
-  #An active class list is the list of class codes that
-  # a user has in the user table.
-  request = {"Key" : {"email": email}, "AttributesToGet": ["classCodes"]}
-  res = dbUtils.getItem(request, table)
-  #Get the class codes for the user.
-  if res is not None and 'Item' in res:
-    classCodes = res['Item']['classCodes']
-
+  classCodes = getClassCodesFromUser(dynamoDBInstance, email)
   #Use the class codes to get the class details for
   # each class.
-  table = dbUtils.getTable('classes', dynamodb)
+  table = dbUtils.getTable('classes', dynamoDBInstance)
   if table is None:
     MentiiLogging.getLogger().error("Unable to get classes table in getActiveClassList")
     return None
```

Comment:
I would change this if block to something of the following
```
if table is None:
  Log shit
  add to response
else
  other stuff
end if

return response
```

ryan mentioned this too